lib/navigation.dart

```diff
@@ -306,6 +306,7 @@ final destinations = [
         label: Text("Freie Zimmer"),
         selectedIcon: Icon(Icons.door_back_door),
         navbarActions: [
+          IconButton(onPressed: setRoomTypeFilterAction, icon: Icon(Icons.tune)),
```

Comment:
```suggestion
          IconButton(onPressed: setRoomTypeFilterAction, icon: Icon(Icons.filter_alt)),
```
Mir gefällt der Trichter besser, kannst du aber auch so lassen.

Reply:
Der wäre eigentlich logischer, fand ich aber zu uneindeutig in dieser Konstellation l.